tools/wrap.start

```diff
@@ -1,5 +1,8 @@
 (function (root, factory) {
-    if (typeof define === 'function' && define.amd) {
+    if (typeof exports === "object"){
```

Comment:
@Beak-man there's a minor formatting irregularity at the end of this line. It should be:
   if (typeof exports === "object") {